### ISSUE TITLE
False positive for TrailingCommaInLiteral with EnforcedStyleForMultiline: comma

### ISSUE THREAD

**Issue by jorhett** [bbatsov/rubocop#3110]
## Expected behavior

No error.

## Actual behavior

```
Rakefile:15:7: C: Put a comma after the last item of a multiline hash.
      tags: ['~FC005', '~FC064', '~FC065']
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

## Steps to reproduce the problem

Enable the following setting in .rubocop.yml
```
Style/TrailingCommaInLiteral:
  EnforcedStyleForMultiline: comma
```

Put a single-line array without a trailing comma inside of a hash assignment

```
testing = {
  tags: ['~FC005', '~FC064', '~FC065']
}
```

Test it with rubocop.

## RuboCop version

Include the output of `rubocop -V`:

```
$ rubocop -V
0.39.0 (using Parser 2.3.1.0, running on ruby 2.1.8 x86_64-darwin14.0)
```

**Comment by mikegee** [comment]
I don't follow. The hash spans across three lines and it says: "Put a comma after the last item of a multiline hash."

Perhaps the array value is confusing things? This produces the same offense:

```ruby
testing = {
  tags: 'FIXME: tags go here'
}
```

```
% bin/rubocop test.rb --only Style/TrailingCommaInLiteral
Inspecting 1 file
C

Offenses:

test.rb:2:3: C: Put a comma after the last item of a multiline hash.
  tags: 'FIXME: tags go here'
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

**Comment by owst** [comment]
I agree, this looks like the correct error to me, there is no trailing comma *after* the array within a hash spanning multiple lines.

**Comment by jorhett** [comment]
Sorry, I meant to close yes. Yes, I misread the complaint as being about the hash entry it highlighted. It would be helpful to change the error message to point at the location where the comma should be added, instead of the entire entry which is not at fault...
